include/seqan3/utility/parallel/all.hpp

```diff
@@ -31,6 +29,6 @@
  */
 //!\endcond
 
- #include <seqan3/utility/parallel/detail/latch.hpp>
- #include <seqan3/utility/parallel/detail/reader_writer_manager.hpp>
- #include <seqan3/utility/parallel/detail/spin_delay.hpp>
+#pragma once
+
+#include<seqan3/core/platform.hpp>
```

Comment:
Empty all.hpp are not valid, but some exists because they only do documentation.
If a all.hpp doesn't include anything, it must include <seqan3/core/platform.hpp>


2. I just notice there is missing a space :-(, Will fix right now

Reply:
Ah okay, I was just confused, that we include some non utility things. THanks for the explanation!